src/main/java/hu/iac/webshop/controllers/DiscountController.java

```diff
@@ -42,13 +43,19 @@ public Discount addDiscount(@RequestBody DiscountRequest discountRequest) {
         return this.discountService.createDiscount(newDiscount);
     }
 
-    @DeleteMapping("/discount/delete/{id}")
-    public void deleteDiscount(@PathVariable long id) {
-        this.discountService.deleteDiscount(id);
+    @DeleteMapping("/discounts/delete/{id}")
+    public ResponseEntity<Long> deleteDiscount(@PathVariable long id) {
+        boolean isRemoved = this.discountService.deleteDiscount(id);
+
+        if (!isRemoved) {
+            return new ResponseEntity<>(HttpStatus.NOT_FOUND);
+        }
+
+        return new ResponseEntity<>(id, HttpStatus.OK);
     }
 
-    @PutMapping("/discount/update/{id}")
-    public ResponseEntity<Discount> updateDiscount(@RequestBody DiscountRequest discountRequest, @PathVariable Long id) {
+    @PutMapping("/discounts/update/{id}")
```

Comment:
Niet perse naar jouw. Maar laten we wel even die endpoints consistent houden. Het is nu overal anders haha.